src/main/java/io/stargate/sgv2/jsonapi/api/model/command/clause/update/ActionTargetLocator.java

```diff
@@ -7,8 +7,17 @@
 import io.stargate.sgv2.jsonapi.exception.JsonApiException;
 import java.util.regex.Pattern;
 
-/** Helper class for resolving "dotted paths" into {@link UpdateTarget} instances. */
-public class UpdateTargetLocator {
+/**
+ * Helper classes built from "dotted paths" and evaluated in context of a JSON document to produce
+ * one of:
+ *
+ * <ul>
+ *   <li>{@link ActionTarget} instances for Document updates
+ *   <li>{@link JsonNode} when extracting value using {@link #findValueIn} (used for Sorting and
+ *       possibly Projection)
+ * </ul>
+ */
+public class ActionTargetLocator {
```

Comment:
Can we have this class named as just TargetLocator and move it to util package? That way we can use it for projection/sorting.

Reply:
@maheshrajamani Yes, I will do that today. Was wondering about it since it's not really update (or update action) specific.

Reply:
Are you going to do it as part of this PR?